Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Filter/CountChildNodesFilter.php

```diff
@@ -79,4 +79,9 @@ public function with(
             $propertyValue ?? $this->propertyValue,
         );
     }
+
+    public function isEmpty(): bool
```

Comment:
one of these unrelated changes that should be their own PR ;)

Reply:
its in a separate commit already https://github.com/neos/neos-development-collection/pull/5512/commits/6ac52fb22d40febd635f25f516bc7947de1d872e i was just onto improving that we also ask the cache if a simple filter is active but didnt continue ...